frontend/public/components/container.tsx

```diff
@@ -48,18 +49,24 @@ const Lifecycle: React.FC<LifecycleProps> = ({ lifecycle }) => {
   const postStart = _.get(fields, 'postStart.cmd');
   const preStop = _.get(fields, 'preStop.cmd');
 
+  const { t } = useTranslation();
+
   const label = (stage: ContainerLifecycleStage) =>
     lifecycle && k8sProbe.getLifecycleHookLabel(lifecycle, stage);
   return (
     <div>
       {postStart && (
         <div>
-          <span>PostStart: {label('postStart')}</span> <code>{postStart}</code>
+          <Trans t={t} ns="public">
+            <span>PostStart: {label('postStart')}</span> <code>{postStart}</code>
```

Comment:
Do you know if this `span` is really needed? It's good to keep the markup inside of `Trans` as simple as possible.

Reply:
I haven't been able to find a container with this lifecycle hook, so I can't say for sure.